robocop/checkers/naming.py

```diff
@@ -203,6 +216,14 @@ def check_keyword_naming(self, keyword_name, node):  # noqa
         if any(word[0].islower() for word in words if word):
             self.report("wrong-case-in-keyword-name", node=node)
 
+    def check_bdd_keywords(self, keyword_name, node):
+        if keyword_name.lower() not in self.bdd:
+            return
+        arg = node.get_token(Token.ARGUMENT)
+        suffix = f". Concatenate it with keyword: '{keyword_name.title()} {arg.value}'" if arg else ""
```

Comment:
...and then here:
```suggestion
        suffix = f". Use one space between: '{keyword_name.title()} {arg.value}'" if arg else ""
```

Reply:
Just a suggestion but still I am thinking about something better and easier to understand.

Reply:
Yeah it's easier to understand, I will go with it